src/components/activity.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 <ion-card data-cy-type="activity" :data-cy-verb="activity.verb">
-  <ion-card-header>
+  <ion-card-header v-if="(objects[0].className != 'Link' && text != '') || objects.length > 1">
```

Comment:
In vue js these are defined on the component. rather than doing the if here, you'd add in `computed`: 
```Typescript
fullView() {
return !( this.text == ''" && this.objects.length == 1 && objects[0].className != 'Link')
}
```
and then here just do
```suggestion
  <ion-card-header v-if="fullView">
```